includes/helper/class-wp-job-manager-helper.php

```diff
@@ -85,11 +93,64 @@ public function init() {
 	 * Initializes admin-only actions.
 	 */
 	public function admin_init() {
+		$this->load_language_pack_helper();
 		add_action( 'plugin_action_links', [ $this, 'plugin_links' ], 10, 2 );
 		add_action( 'admin_notices', [ $this, 'licence_error_notices' ] );
 		$this->handle_admin_request();
 	}
 
+	/**
+	 * Load the language pack helper.
+	 */
+	private function load_language_pack_helper() {
+		if ( $this->language_pack_helper ) {
+			return;
+		}
+
+		$this->language_pack_helper = new WP_Job_Manager_Helper_Language_Packs( $this->get_plugin_versions(), $this->get_site_locales() );
+	}
+
+	/**
+	 * Get the versions for the installed managed plugins, keyed with the plugin slug.
+	 *
+	 * @return string[]
+	 */
+	public function get_plugin_versions() {
+		return array_filter(
+			array_map(
+				function( $plugin ) {
+					return $plugin['Version'];
+				},
+				$this->get_installed_plugins( false )
+			)
+		);
+	}
+
+	/**
+	 * Get the locales used in the site.
+	 *
+	 * @return string[]
+	 */
+	private function get_site_locales() {
+		$locales = array_values( get_available_languages() );
+
+		/** This action is documented in WordPress core's wp-includes/update.php */
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
+		$locales = apply_filters( 'plugins_update_check_locales', $locales );
+		$locales = array_unique( $locales );
+
+		return $locales;
+	}
+
+	/**
+	 * Get the language packs helper.
+	 *
+	 * @return WP_Job_Manager_Helper_Language_Packs
+	 */
+	public function get_language_pack_helper() {
```

Comment:
Are we using this method?

Reply:
I meant to add a review comment. Since this isn't a singleton, I wanted to ensure people were able to remove the filter added in the class by accessing the instance. Alternatively, we could apply a filter in the class that would allow people to disable this functionality.

Reply:
For now, I got rid of it and added the filter in d3fd874b8d380147867ecba35c059778907347a5 that makes it a bit easier if someone wanted to disable this functionality.